frontend/src/types/CamperTypes.ts

```diff
@@ -77,3 +79,5 @@ export type WaitlistedCamper = {
   campSession: string;
   linkExpiry?: Date;
 };
+
+export type CheckoutUrl = { checkoutSessionUrl: string };
```

Comment:
why does this need to be an object?

Reply:
I guess it doesn't really, more of nice to have to see mark the expected response structure. Maybe renaming to `CheckoutUrlResponse` makes more sense?

Reply:
yeah I think that would be clearer